### PR TITLE
feat(fluent-bit): add Dockerfile for Fluent Bit

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
image pushed to test: ghcr.io/kkujawa-sumo/fluent-bit:v1.7.2-ubi-test

### CODE REVIEW

File: fluent-bit/v1.7.2/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      version="1.7.2" \
+      release="1" \
+      summary="UBI based Fluent Bit" \
+      description="Fluent Bit is a fast Log Processor and Forwarder"
+
+RUN microdnf update && microdnf install -y openssl libpq systemd && microdnf clean all
+COPY --from=builder /fluent-bit /fluent-bit
+
+RUN mkdir licenses && \
+    mv /fluent-bit/LICENSE /licenses/
```
Comment (sumo-drosiek) [inline fluent-bit/v1.7.2/Dockerfile:48]:
```suggestion
COPY /fluent-bit/LICENSE /licenses/LICENSE
```

Comment (kkujawa-sumo) [reply fluent-bit/v1.7.2/Dockerfile:48]: Directory must exists to copy single file to the directory I could write something like this:
```
RUN mkdir /path/to/your/new/folder/
COPY /host/folder/* /path/to/your/new/folder/
```
but I prefer existing form

Comment (sumo-drosiek) [reply fluent-bit/v1.7.2/Dockerfile]: Could you try with ADD?
```
ADD https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/b86ab77239f2a11ee69ad05b24122958d8b2df5b/LICENSE \
    /licenses/LICENSE
```

**Review by sumo-drosiek — APPROVED** [review]

File: fluent-bit/v1.7.2/Dockerfile (hunk 1 above)
Comment (kkujawa-sumo) [reply fluent-bit/v1.7.2/Dockerfile]: with `ADD` it works and I can also copy licence from builder to new directory without creating it

Comment (sumo-drosiek) [reply fluent-bit/v1.7.2/Dockerfile]: It's fine now :+1:

**Pull request merged**